docs/install-olm.md

```
@@ -14,10 +14,10 @@ This document will walk you through setting up the required configuration to ins
 
 5. [Authorino Resilient Configuration](#authorino-topologyconstraints-poddisruptionbudget-and-resource-limits)
 
-4. [[OpenShift Specific] Setup Observability ](#set-up-observability-openshift-only)
+6. [[OpenShift Specific] Setup Observability ](#set-up-observability-openshift-only)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)